### ISSUE TITLE
Does it support text detection?

### ISSUE THREAD

**Issue by zcswdt** [ultralytics/yolov5#3915]

**Comment by glenn-jocher** [comment]
YOLOv5 supports training on any labelled object detection dataset. See **Train Custom Data** tutorial to get started.

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [TorchScript, ONNX, CoreML Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)

**Comment by Sushant-ctrl** [comment]
YOLO can be trained to detect text but Its better to use an OCR

**Comment by zcswdt** [comment]
> YOLO can be trained to detect text but Its better to use an OCR

Because I want to do video text detection, I saw that yolov5 can process video, so I thought of using this project to do video text detection, I don’t know if it’s possible

**Comment by zcswdt** [comment]
> YOLOv5 supports training on any labelled object detection dataset. See **Train Custom Data** tutorial to get started.
> 
> ## YOLOv5 Tutorials
> * [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)  🚀 RECOMMENDED
> * [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)  ☘️ RECOMMENDED
> * [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)  🌟 NEW
> * [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)  🌟 NEW
> * [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
> * [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)  ⭐ NEW
> * [TorchScript, ONNX, CoreML Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
> * [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
> * [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
> * [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
> * [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
> * [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)  ⭐ NEW
> * [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)

Thank you very much. I want to change the code during text detection, which is to divide each box into many small boxes during training, and then use these small boxes to train. After the training is completed, I will merge these small boxes to complete Text detection. I wonder if the various data enhancements used by Yolov5 during training will affect my operation?

**Comment by pderrenger** [comment]
You're welcome! YOLOv5's data augmentations could affect your approach, as they modify images and labels during training. You might need to customize the augmentation pipeline to ensure it aligns with your method of dividing and merging boxes. Check the `datasets.py` file for augmentation details.
